Client.xcodeproj/project.pbxproj

```diff
@@ -474,6 +474,7 @@
 		D83822001FC7961D00303C12 /* DispatchQueueExtensions.swift in Sources */ = {isa = PBXBuildFile; fileRef = D83821FF1FC7961D00303C12 /* DispatchQueueExtensions.swift */; };
 		D863C8F21F68BFC20058D95F /* GradientProgressBar.swift in Sources */ = {isa = PBXBuildFile; fileRef = D863C8E31F68BFC20058D95F /* GradientProgressBar.swift */; };
 		D87F84AC20B891160091F2DA /* TabDisplayManager.swift in Sources */ = {isa = PBXBuildFile; fileRef = D87F84AB20B891160091F2DA /* TabDisplayManager.swift */; };
+		D88C01B722B29EC200936951 /* ScreenGraphTest.swift in Sources */ = {isa = PBXBuildFile; fileRef = 39012F271F8ED262002E3D31 /* ScreenGraphTest.swift */; };
```

Comment:
@farhanpatel I'm not sure why this test suite has been added to the L10nSnapshotTests, there is not any call in those tests to take screenshots while running them...

Reply:
There are a few extensions that are added in that file. It looks like it is required in order to run some of the tests

Reply:
Oh I see, okay...that would be fine then. But what worries me is the change commented below about changing the graph that will break the regular tests.